content/docs/user-guide/assets/asset-processor/interface.md

```diff
@@ -47,33 +46,44 @@ Jobs in the Asset Status list display the statuses below:
+Completed with errors produces a product asset but it may not be valid for all use cases, such as having incomplete data. Errors happen when data required for the product asset is missing or cannot be produced. Using the example above, suppose the `.assetinfo` specifies tangents should be generated for the mesh product assets, but the Asset Builder fails to produce the tangents. The end result is a valid product asset that does not contain all the requested data.
 
-Failed with errors does not produce a product asset.
+Failed jobs do not produce a product asset.
+
+What is considered a log message, warning, error, and failure, and how the system handles these is up to each builder's author. Our guidance is:
+ * Print a log message for anything that will be useful as a bread crumb for tracking down problems later, for both the content creator or the builder author.
+ * Emit a warning if something is wrong but it can mostly be handled, and the output your builder can generate is reasonably close to what the content creator intended.
+ * Post an error if your builder encounters data it can't properly handle and you know the product asset you output won't be fully correct. When possible, this is prefered over a failure because the failure won't produce any product asset, so any references from other assets to this asset may break. For example, if a prefab references a mesh product asset from an FBX file, an update to the FBX causes the job to fail, then any edits to the prefab while this job is failing may lose the reference to the mesh.
+ * Post a failure if your builder encounters something it cannot happen and all processing should end, and the builder should not output anything. This is generally considered a last resort, and is most commonly seen when a builder crashes during processing.
 
 {{< note >}}
-The most common reason for failure is that the Asset Builder has crashed. When an Asset Builder crashes, Asset Processor logs the job failure with an error and continues processing assets from the jobs list. When Asset Processor is restarted, it attempts to reprocess any failed jobs. Alternatively, you can locate individual source assets in the Assets tab, **right-click** on the asset, and select **Reprocess File** to re-run the process job.
+The most common reason for failure is that the Asset Builder has crashed. When an Asset Builder crashes, Asset Processor logs the job failure with a failure and continues processing assets from the jobs list. When Asset Processor is restarted, it attempts to reprocess any failed jobs. If an asset has failed to process, we recommend you first examine why this failure occured, and inform your team, sharing any relevant artifacts such as log messages. Once you proceed to the next step, that information may be lost. Alternatively, you can locate individual source assets in the Assets tab, **right-click** on the asset, and select **Reprocess File** to re-run the process job.

```

Comment:
"logs the job failure with a failure" is kind of confusing.  Perhaps: "logs the job as failed"

Reply:
Fixed